Pizza.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Chocolate Pizza</title>
+  <link rel="stylesheet" href="pizza.css">
+  <!-- <link rel = "stylesheet" href = "pizza.css" type = "text/css"/> -->
+  <link href = "http://www.myfontfree.com/brush-script-mt-myfontfreecom126f37001.htm"/>
+  <link href="https://fonts.googleapis.com/css?family=Anton" rel="stylesheet"/>
+</head>
+
+<body>
+<header>
+
+<img src = "img/logo.png"/>
+<div class="head-text">
+  <span class ="delicious">Delicious</span><br />
+  <span class="best">THE BEST FOOD BLOG ON THE WEB</span>
+</div>
+<div id = "social-media">
+  <ul>
+    <li><img src = "img/fb-icon.png" alt = "facebook"/></li>
+    <li><img src = "img/twit-icon.png"alt = "twitter"/></li>
+    <li><img src = "img/gp-icon.png" alt = "google"/></li>
+    <li><img src = "img/insta-icon.png" alt = "instagram"/></li>
+    <li><img src = "img/flic-icon.png" alt = "flicker"/></li>
+    <li><img src = "img/pint-icon.png" alt = "pinterest"/></li>
+    <li>&nbsp</li>
+    <li><img src = "img/rss-icon.png" alt = "really simple syndication"/></li>
+    <li><img src = "img/mail-icon.png" alt = "mail"/></li>
+  </ul>
+</div>
+</header>
+<div class="separator">
```

Comment:
Make sure to be attentive to indentation in your html and css files, just like you would be in your js files!